lib/process.ts

```diff
@@ -0,0 +1,49 @@
+import { FarmerJournalSchema } from "@/components/forms/farmer-journal/schema";
+import { TechnicianVisitFormSchema } from "@/components/forms/technician-visit/schema";
+import { FarmerJournal } from "@/types/content";
+import { CaseType, serialize } from "jsonapi-fractal";
+
+type FarmerJournalSerializedType = FarmerJournal & {
+  title: string,
+}
+
+export const processJournal = (data: FarmerJournalSchema) => {
+  const fieldTotalmortality = data.fieldMortality + data.fieldMortalityprolapse;
+  const totalEggs = data.fieldSmallEggs = data.fieldMediumEggs + data.fieldLargeEggs;
+  const now = new Date();
+  const title = `Daily journal ${now.getFullYear()}-${now.getMonth() + 1}-${now.getDate()}`;
+
+  const payload: FarmerJournalSerializedType = {
+    ...data,
+    title,
+    fieldTotalmortality,
+    fieldClosingStock: data.fieldInitialstock - fieldTotalmortality,
+    fieldMortalityPercentage_: data.fieldInitialstock ? fieldTotalmortality / data.fieldInitialstock * 100 : undefined,
+    fieldProducedEggs: totalEggs - data.fieldDamagedEggs,
+    // FIXME: trailing underscores get ignored by the serializer
+    // fieldDamagedEggsPercentage__: totalEggs ? data.fieldDamagedEggs / totalEggs * 100 : undefined,
```

Comment:
Oh shoot, should I change this in the backend? It would delete all the data in the backend, but I don't think we have anything too meaningful right now anyways.

(I can also reimport Warren's data)